src/main/resources/i18n/messages.properties

```diff
@@ -78,6 +78,7 @@ ranked1v1.play=Play
 ranked1v1.cancel=Cancel
 ranked1v1.choseFactions.title=Select factions
 ranked1v1.choseFactions.description=When the match starts, one of your selected factions will be assigned to you.
+ranked1v1.queuePopTimer=Next round in {0,number,0}:{1,number,00}
```

Comment:
Isn't it "next matching" ?

Reply:
Maybe thats better, gonna ask around a bit. Gonna put that in if nobody objects.

Reply:
So we (kinda) settled on `Matching in x:xx` without the "Next".